lib/electric_slide/call_queue.rb

```diff
@@ -62,7 +62,7 @@ def available_agent_summary
     # @return {Agent}
     def checkout_agent
       agent = @strategy.checkout_agent
-      agent.presence = :busy
+      agent.presence = :busy if agent
```

Comment:
I hear your point. I will try again and see if I can recreate this issue, but so far I have been unsuccessful at small scale. Only happens in larger tests. 

The add_agent method is protected on both sides (my application side and ES) so the error is elsewhere. Will get back to you if I find anything.